src/components/ui/billing-dialog/components/billing-info.tsx

```
@@ -0,0 +1,116 @@
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Badge } from "@/components/ui/badge";
+import { Separator } from "@/components/ui/separator";
+import { getPlanConfig, type PlanId } from "@/lib/config/plans";
+import type { BillingInfo } from "@/lib/services/billing";
+
+type BillingInfoProps = {
+  billingInfo: BillingInfo | null;
+  isLoading?: boolean;
+};
+
+export function BillingInfo({ billingInfo, isLoading }: BillingInfoProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'BillingInfo'. Consider to delete it or rename it.

'BillingInfo' is defined here:


(lint/suspicious/noRedeclare)